apps/molgenis-components/src/components/forms/InputText.vue

```diff
@@ -78,12 +98,12 @@ export default {
 </template>
 <script>
   export default {
-    data: function () {
+    data: function() {
       return {
         value: null,
-        value2: "this is a default value"
+        value2: 'this is a default value',
       };
-    }
+    },
   };
```

Comment:
Why does this use different styling than the code inside the upper <script> ?

Reply:
Yeah, it is time for automatformatting hook before commit.